projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html

```diff
@@ -14,69 +14,93 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<div class="container px-0 mt-1 mb-2" *ngIf="item && permissions">
-  <div id="item-{{ item.metadata.barcode }}" class="row">  <!-- First row :: item detail -->
-    <div class="col-sm-3">
-      <admin-record-masked *ngIf="item.metadata._masked" [record]="item"></admin-record-masked>
+<div class="container position-relative item px-0 mt-1 mb-2" *ngIf="item && permissions">
+  <!-- METADATA -->
+  <div class="row">
+    <!-- Barcodes -->
+    <div class="col-3 pl-5 font-weight-bold label-title" translate>Barcode</div>
```

Comment:
Can the font-weight as a css property of label-title?

Reply:
nope : the purpose of `label-title` seems to add a ':' characrter after the label, not to set the font-weight to bold.